presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -491,4 +492,17 @@ public PrestoSparkConfig setNativeExecutionBroadcastBasePath(String nativeExecut
         this.nativeExecutionBroadcastBasePath = nativeExecutionBroadcastBasePath;
         return this;
     }
+
+    public boolean isNativeTriggerCoredumpWhenUnresponsiveEnabled()
+    {
+        return nativeTriggerCoredumpWhenUnresponsiveEnabled;
+    }
+
+    @Config("native-trigger-coredump-when-unresponsive-enabled")
```

Comment:
Nit: name this something like `native-kill-unhealthy-native-process` ?
Triggering coredump is meta specific behavior/infra

Reply:
The application will deliver `SIGSYS` when this functionality is enabled. This signal should trigger a coredump on any linux system run on ARM / x86: https://man7.org/linux/man-pages/man7/signal.7.html

Reply:
Ah i see. Makes sense. Thanks!